tests/netlify-deploy.ts

```diff
@@ -133,7 +124,7 @@ export class NextDeployInstance extends NextInstance {
 
     const deployRes = await execa(
       'npx',
-      ['netlify', 'deploy', '--build', '--json', '--message', deployTitle ?? ''],
```

Comment:
there's a weird bug in `ntl deploy --json` that makes debugging failures annoying anyway so I like this change :)

(I should write it up but I don't know how to reproduce it 😞)

Reply:
I think we are just not getting any output at all on failures with `--json` flag. At least that's was my experience here and why I wanted to change this stuff in the first place to see actual logs on deploy failures

Reply:
yeah, I think that's right... maybe it's by design?